lib/samples/show_wfs_layer_with_xml_query/show_wfs_layer_with_xml_query.dart

```diff
@@ -0,0 +1,122 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() async {
+  // Supply your apiKey using the --dart-define-from-file command line argument
+  ArcGISEnvironment.apiKey = const String.fromEnvironment('API_KEY');
+
+  runApp(
+    const MaterialApp(
+      home: ShowWfsLayerWithXmlQuery(),
+    ),
+  );
+}
+
+class ShowWfsLayerWithXmlQuery extends StatefulWidget {
+  const ShowWfsLayerWithXmlQuery({super.key});
+
+  @override
+  State<ShowWfsLayerWithXmlQuery> createState() =>
+      _ShowWfsLayerWithXmlQueryState();
+}
+
+class _ShowWfsLayerWithXmlQueryState extends State<ShowWfsLayerWithXmlQuery> {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
```

Comment:
We don't need a SafeArea when the sample is just a MapView without any controls. For example, see display_annotation.dart.